app/_locales/es/messages.json

```diff
@@ -86,7 +86,7 @@
     "message": "Aprobado"
   },
   "attemptingConnect": {
-    "message": "Intentando conectar a la Blockchain"
+    "message": "Intentando conectar a la Cadena de Bloques"
```

Comment:
I wouldn't translate "Blockchain".

Reply:
Yeah, I don't care really one way or the other. I just noticed we hadn't been consistent with it previously, so I made them all the same.